pymc/tests/distributions/test_transform.py

```diff
@@ -598,3 +615,31 @@ def test_discrete_trafo():
         with pytest.raises(ValueError) as err:
             pm.Binomial("a", n=5, p=0.5, transform="log")
         err.match("Transformations for discrete distributions")
+
+
+def test_transforms_ordered():
```

Comment:
Perhaps a more direct name?

```suggestion
def test_2d_univariate_ordered_():
```